src/ocean/task/Scheduler.d

```diff
@@ -334,6 +334,7 @@ final class Scheduler : IScheduler
             auto caller_task = Task.getThis();
             if (caller_task !is null)
                 caller_task.kill();
+            return;
```

Comment:
Unrelated to this PR, but it might be worth adding a comment as to why the calling task is killed if `schedule` is called during shutdown.

Reply:
Will add